go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/tidwall/gjson v1.17.0
 	golang.org/x/crypto v0.11.0 // indirect
-	golang.org/x/exp v0.0.0-20230711153332-06a737ee72cb
```

Comment:
How did this happen?

Did you `go mod tidy`?

Reply:
yes I ran go mod tidy which added it